src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Application app = new Application();
+        for (int i = 1; i < 4; i++) {
```

Comment:
The loop condition `i < 4` is using a magic number. Consider defining a constant that represents the number of balls you want to generate, which will make the code more understandable.